lua/entities/gmod_wire_expression2/core/custom/sv_rune2.lua

```diff
@@ -2,24 +2,6 @@ local table_remove = table.remove
 local table_insert = table.insert
 local luaTableToE2, getE2UDF = vex.luaTableToE2, vex.getE2UDF
 
-
--- Builds a body to run an e2 udf and pass args to it.
-local function buildBody(args) -- Why does the wireteam do this
```

Comment:
Shouldn't have removed this. I left this there because for the vararg support for the `try` in the next PR.

Reply:
I didn't remove it, it's in vex now